ftl/core/browsing.ftl

```diff
@@ -28,6 +28,7 @@ browsing-change-note-type2 = Change Note Type...
 browsing-change-notetype = Change Notetype
 browsing-clear-unused-tags = Clear Unused Tags
 browsing-confirm-saved-search-overwrite = A saved search with the name { $name } already exists. Do you want to overwrite it?
+browsing-create-copy = Create Copy...
```

Comment:
Minor point: the '...' is convention on desktop apps, but seems to be uncommon on mobile, so by using it we're sort of creating desktop-only strings. Not sure what the best way is to handle this - we could put the string in desktop/ instead, or create one string that references the other - but I fear that may end up being more complicated for the translators then just asking them to translate the string twice.

(the instances above this one were imported from the old translations)

Reply:
What if we had a

```
# [verbose explanation]
new-window-action = { $action }...
...
browsing-create-copy = Create Copy
```

and a util that takes an action name and wraps it in `tr.new_window_action(action=action_name)` if appropriate?

Reply:
That's a good idea :-)